website/app/styles/doc-components/algolia-search/index.scss

```
@@ -10,7 +10,9 @@
 @import "./results";
 
 // "autocomplete" target container - not used
-// .doc-algolia-search__autocomplete-container {}
+.doc-algolia-search__autocomplete-container {
+  position: absolute;
```

[suggestion]
```suggestion
  // we assign it a position absolute to remove it from the layout flow, so the `gap` property of the parent flex container doesn't get applied to this as well
  position: absolute;
```

---

[suggestion] a possible alternative approach to solving the problem
```suggestion
  // we use this trick so the `gap` property of the parent flex container doesn't get applied to this as well
  display: contents;
```

---

a good alternative, indeed. not sure which one is 'better', so I'll stick with `position: absolute`